doc/src/manual/arrays.md

```diff
@@ -945,6 +945,16 @@ julia> string.(1:3, ". ", ["First", "Second", "Third"])
  "3. Third"
 ```
 
+Sometimes, you have a non-scalar object like an array that you wish to treat as a scalar in a particular
```

Comment:
Here's an alternative that avoids "scalar" entirely and aims at describing the _why_ instead of just prescribing the _what_.  This could surely be improved upon, but I think I favor talking about this invertedly — that is, "protecting from iteration" instead of "treating like scalar."

> Sometimes, you want a container (like an array) that would normally participate in broadcast to be "protected" from broadcast's behavior of iterating over all of its elements.  By placing it inside _another_ container (like a one-tuple) broadcast will treat it as a single value.

Reply:
Yep, that makes sense.